xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -1010,6 +1020,11 @@ void ResizeWindowInternal(int newWidth, int newHeight, int newLeft, int newTop,
     [windowedFullScreenwindow update];
     [blankView setFrameSize:NSMakeSize(m_nWidth, m_nHeight)];
 
+    // it seems, that in macOS 10.15 this defaults to YES, but we currently do not support
+    // Retina resolutions properly. Thus, ensure that we get a 1 pixel per point framebuffer.
+    if ([blankView respondsToSelector:@selector(setWantsBestResolutionOpenGLSurface:)])
```

Comment:
there's no need for the `-respondsToSelector:` check - it always evaluates to `YES`, as `wantsBestResolutionOpenGLSurface` property is available since macOS 10.7 and Kodi's deployment target is 10.9.

Replace these 2 lines to just `blankView.wantsBestResolutionOpenGLSurface = NO;`

Reply:
Thanks, will do.